src/orquestra/sdk/_ray/_ray_logs.py

```diff
@@ -113,7 +115,7 @@ def iter_task_logs(
             )
 
 
-class DirectRayReader:
+class DirectLogReader:
```

Comment:
Not really a blocker, this conversation doesn't affect how the system behaves. Normally I'd encourage spending a moment or two to discuss our strategies for code organization, because team alignment comes with a compounding interest (or losses!). Maybe a topic for the next coffee hour cc @zapatacomputing/sdk-team?

Reply:
Let me add a ticket to the inbox for this as well, if we decide not to act we can decline